components/images/images.css

```diff
@@ -0,0 +1,36 @@
+.img-100pr {
+    width: 100%;
+}
+
+.img-75pr {
+    width: 75%;
+}
+
+.img-50pr {
+    width: 50%;
+}
+
+.img-25pr {
+    width: 25%;
+}
+
+.img {
```

Comment:
Looks good. Is there any specific use case of this class with fixed height and width?

Reply:
yes. For the display of image in that gray box on component page.  I will change it to max-width. thanks for pointing out.